script/build-rules.js

```diff
@@ -567,11 +566,29 @@ presets(root).then((presetObjects) => {
 
       let first = true
       /** @type {string} */
-      let setting
-
-      for (setting in tests) {
-        if (own.call(tests, setting)) {
-          const fixtures = tests[setting]
+      let configuration
+
+      for (configuration in tests) {
+        if (own.call(tests, configuration)) {
+          /** @type {{settings: Record<string, unknown>, rawOptions: unknown}} */
+          const {settings, rawOptions} = JSON.parse(configuration)
+          const fixtures = tests[configuration]
+
+          // Don't show the same test for both shared settings and rules' options.
```

Comment:
- [x] I've hidden repetitious examples from the readmes:
- All of the existing tests are retained.
- Each readme has the same number of examples as it does now.